util/fibers/epoll_socket.cc

```diff
@@ -426,7 +436,7 @@ bool EpollSocket::SuspendMyself(detail::FiberInterface* cntx, std::error_code* e
 void EpollSocket::Wakey(uint32_t ev_mask, int error, EpollProactor* cntr) {
   DVSOCK(2) << "Wakey " << ev_mask;
 #ifdef __linux__
-  constexpr uint32_t kErrMask = EPOLLERR | EPOLLHUP;
+  constexpr uint32_t kErrMask = EPOLLERR | EPOLLHUP | EPOLLRDHUP;
```

Comment:
was there a reason for EPOLLRDHUP ? I think we got the event with `EPOLLHUP` as well.

Reply:
I get a `0x2005` aka `EPOLLIN | EPOLLOUT | EPOLLRDHUP` mask when the socket closes.